README.md

```diff
@@ -1,2 +1,55 @@
-# Hello-Microverse
-In this project i am going to learn about the git tool and build a simple project by using HTML and CSS.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is the simple project I learn about the git command and building a simple page
+
+
+## Built With
+
+- HTML and CSS
+- Git and github
+
+## Live Demo (if available)
+There no live demo available.
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Please make sure you are up to date if you have been working on this project. If you are new just simply clone the repository.
+### Setup
+There is not specific setup needed for this project
+### Install
+Ther are no specific installations needed for this project apart from your code editor and git bash.
+### Usage
+Anyone can use this project by giving proper credit.
+
+## Authors
+
+👤 **Shaheryar abid**
+
+- GitHub: [@githubhandle](https://github.com/Shaheryar0054)
+- Twitter: [@twitterhandle](https://twitter.com/sharyar0310)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shaheryar-abid-8758121b3/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] Kindly add "Microverse" in the acknowledgments. 👆🏽

Reply:
okay